smartseq2_single_sample/tasks/hisat2.wdl

```diff
@@ -0,0 +1,78 @@
+task HISAT2PE {
+  File hisat2_ref
+  File fq1
+  File fq2
+  String ref_name
+  String output_name
+  
+  command {
+    tar -zxvf "${hisat2_ref}"
+    /opt/tools/hisat2-2.1.0/hisat2 -t \
```

Comment:
If any of these commands (-t, -x, -1, -2, -p) have long-form arguments, I've been using those instead of the short forms in wdls whenever I can because they're more expressive and easier for a user to read/easily reuse.

Reply:
I looked into the manual. for '-t, -x, -1, -2, -p', there are only short name available